### ISSUE TITLE
NOAA3 icon display issues when other modules are enabled

### ISSUE THREAD

**Issue by bhaules** [cowboysdude/MMM-NOAA3#44]
I’ve been using this module for a while now, and it’s been great.
I have recently installed a new module for solar panels and somehow it’s affecting the NOAA module.
When the Solar module is innactive the NOAA module looks okay:
![2020-04-08 08_23_44-Window](https://user-images.githubusercontent.com/45819454/78741708-a3e07500-798c-11ea-9bf5-0e285199875c.png)

When the solar module is active the NOAA icons look weird.
![2020-04-08 08_22_43-Window](https://user-images.githubusercontent.com/45819454/78741698-99be7680-798c-11ea-8689-0557fd59a1c3.png)

Any thoughts on how I can fix this? And why is it happening?

HB

**Comment by cowboysdude** [comment]
It's a css issue.   Try a different css file.......noaa2.css or noaa1.css, etc.

**Comment by bhaules** [comment]
I have changed it to NOAA1 and NOAA2 css and I'm getting the same issue :(
![image](https://user-images.githubusercontent.com/45819454/80173857-aba64780-8623-11ea-9cae-671c0d325f5f.png)
Tried looking through the .css myself but I'm not skilled enough to figure out what is the issue. I have changed some align-content: center to "left/right" to see if that fixes it but it didn't fix the issue.
I can absolutely live with this and I love this module a lot for the information it gives. It's just something that sits on my mind and couldn't figure out how to do it myself.

HB
